Boilerplate code for frontend and backend integration

kavitha-subr opened the pull request:
Added a basic template for the frontend
Added integration with backend for a single request and response.
Added cors policy for backend code.

Stibreant reviewed (approved):
LGTM. I have one remark, I don't understand why we changed the `.gitignore` It does the same as before, as far as I can tell. IMO it's a little harder to see that we only want to include `package.json` in *-app folders, the other two line changes are fine. Anyway, it does the same, so no reason to nitpick

kavitha-subr commented:
> LGTM. I have one remark, I don't understand why we changed the `.gitignore` It does the same as before, as far as I can tell. IMO it's a little harder to see that we only want to include `package.json` in *-app folders, the other two line changes are fine. Anyway, it does the same, so no reason to nitpick

So, we do want all contents in the app folder except the out folder, that's why I changed that.
